csrc/ir/internal_base_nodes.h

```diff
@@ -229,6 +229,10 @@ class IterDomain : public Val {
     return (isBlockDim() || isThreadDim());
   }
 
+  bool isDevice() const {
```

Comment:
To make the naming consistent as `isThreadDim`, `isBlockDim`:

```suggestion
  bool isDeviceDim() const {
```

Reply:
I renamed to isDeviceDim()

About renaming `isThread` to `isParallelized` I think it makes sense. 

What do you mean though by "include `isBlockDim`"? Did you rather mean  "include `isDeviceDim`" ? If yes, I agree with you in principle, but on the other hand I find it safer for now to just add isDeviceDim() wherever needed

Reply:
Sorry, yes, I meant `isDeviceDim`. As you needed to change some parts of the conditions to include both `isThread()` and `isDeviceDim()`, if the former included `isDeviceDim`, it didn't need any change. We would need to review all the uses of `isThread`, so I think it's fine for now, but something to consider later.